v2.0/create-sequence.md

```diff
@@ -4,13 +4,18 @@ summary:
 toc: false
 ---
 
-<span class="version-tag">New in v2.0:</span> The `CREATE SEQUENCE` [statement](sql-statements.html)...
+<span class="version-tag">New in v2.0:</span> The `CREATE SEQUENCE` [statement](sql-statements.html) creates a new sequence in a database. Use a sequence to auto-increment integers in a table.
 
 <div id="toc"></div>
 
+## Considerations
+
+- Using a sequence is slower than using the `SERIAL` data type– incrementing a sequence requires a write to persistent storage, whereas generating a new `SERIAL`value does not. Use the `SERIAL` data type unless a sequence is preferred or required.
```

Comment:
nit: "type - increment" Missing space between type and -

Reply:
When will a sequence be required? Can we provide example use-cases?

Reply:
1. i think, technically, the em dash doesn't need a space on either side / it depends on the style guide we're adhering to. that said, i added in the space :)
2. sometimes, an app might require using a sequence instead of a randomized int. i'm not sure of a specific example - @vilterp can you provide one?